web/impact/impact/graphql/expert_profile.py

```diff
@@ -0,0 +1,29 @@
+import graphene
+from graphene_django import DjangoObjectType
+from accelerator.models.expert_profile import ExpertProfile
+from impact.graphql.user import UserType
+from impact.graphql.industry import IndustryType
+from impact.graphql.program_family import ProgramFamilyType
+from impact.graphql.startup import StartupType
+
+
+class ExpertProfileType(DjangoObjectType):
+    mentees = graphene.List(StartupType)
+
+    class Meta:
+        model = ExpertProfile
+        only_fields = (
```

Comment:
Is the use of whitelist vs blacklist an intentional decision about the semantics of the two approaches? 
I note that exclude_fields would be a much shorter list.

Reply:
I generally prefer whitelists due to the default behavior of adding a new attribute. Forgetting to whitelist an attribute means it doesn't appear in our API. Forgetting to blacklist a sensitive attribute could lead to unforeseen consequences.